src/Psalm/IssueBuffer.php

```diff
@@ -554,12 +555,13 @@ function (IssueData $d1, IssueData $d2) : int {
 
             foreach ($after_analysis_hooks as $after_analysis_hook) {
                 /** @psalm-suppress ArgumentTypeCoercion due to Psalm bug */
```

Comment:
Is this suppression still required?

Reply:
yeah. FYI, the error is `Argument 2 of Psalm\Plugin\Hook\TransferObject\AfterAnalysisEvent::__construct expects array<string, list<Psalm\Internal\Analyzer\IssueData>>, parent type array<string
, array<list-key<$file_issues>, Psalm\Internal\Analyzer\IssueData>> provided`